accountsdb-plugin-manager/src/accountsdb_plugin_service.rs

```diff
@@ -170,13 +170,24 @@ impl AccountsDbPluginService {
         let libpath = result["libpath"]
             .as_str()
             .ok_or(AccountsdbPluginServiceError::LibPathNotSet)?;
+        let mut libpath = PathBuf::from(libpath);
```

Comment:
Can you run cargo fmt and clippy? CI complains of format issues. thanks!

Reply:
Sorry, ran both now and amended